src/app/Fake.Core.Variables/Variables.fs

```diff
@@ -21,29 +25,35 @@ let getOrDefault<'a> name defaultValue =
 let remove name =
   forceFakeContext()
   |> removeFakeContext name
-  |> Option.map (fun o -> o :?> 'a)
+  |> ignore
```

Comment:
Sometimes you might want to know which value has been removed? Anyway I'm not sure if it is worth the potential breaking change ;) (On the other hand this one isn't too bad - I don't think anyone uses this directly at the moment.

Reply:
I was working on this being brand new, so not breaking change.

Reply:
Ah sorry, I'm probably a bit over cautious on this thing :)
Problem is this is the "connection" between runtime and script and breaking stuff there is a huge pain to cleanup :P

Reply:
I completely agree, I was unaware of the link originally